docs/release-notes-pxc-v8.0.32.md

```diff
@@ -0,0 +1,65 @@
+
+Improvements and bug fixes introduced by Oracle for MySQL 8.0.32 and included in Percona XtraDB Cluster and Percona Distribution for MySQL are the following:
+
+* A replica can add a Generated Invisible Primary Keys(GIPK) to any InnoDB table. To achieve this behavior, the `GENERATE` value is added as a possible value for the `CHANGE REPLICATION SOURCE TO` statement’s `REQUIRE_TABLE_PRIMARY_KEY_CHECK` option.
+
+* The `REQUIRE_TABLE_PRIMARY_KEY_CHECK = GENERATE` option can be used on a per-channel basis.
+
+* Setting `sql_generate_invisible_primary_key` on the source is ignored by a replica because this variable is not replicated. This behavior is inherited from the previous releases.
+
+* An upgrade from 8.0.28 caused undetectable problems, such as server exit and corruption.
+
+* A fix for after an upgrade, all columns added with `ALGORITHM=INSTANT` materialized and have `version=0` for any new row inserted. Now, a column added with `ALGORITHM=INSTANT` fails if the maximum possible size of a row exceeds the row size limit, so that all new rows with materialized `ALGORITHM=INSTANT` columns are within row size limit. (Bug #34558510)
+
+* After a drop, adding a specific column using the INSTANT algorithm could cause a data error and a server exit. (Bug #34122122)
+
+* An online rebuild DDL no longer crashes after a column is added with `ALGORITHM=INSTANT`. Thank you Qingda Hu for reporting this bug. (Bug #33788578, Bug #106279)
+
+Find the full list of bug fixes and changes in the [MySQL 8.0.32 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-32.html).
+
+## Supplied components
+
+Review each component’s release notes for What’s new, improvements, or bug fixes. The following is a list of the components supplied with the Percona XtraDB Cluster-based variation of the Percona Distribution for MySQL:
+
+| Component               | Version   | Description                                |
+| ----------------------- | --------- | -------------------------------------------|
+| Percona XtraBackup      | [8.0.32-26](https://docs.percona.com/percona-xtrabackup/8.0/release-notes/8.0/8.0.32-26.0.html)| An open-source hot backup utility for MySQL-based servers that doesn’t lock your database during the backup.|
+| HAProxy                 | [2.6.12](https://git.haproxy.org/?p=haproxy-2.6.git;a=commit;h=f5884628ed25777fbffe9e3f097ffeff60e1fd1c) | A high-availability and load-balancing solution for Percona XtraDB Cluster. This is a default proxy.|
+| ProxySQL                | [2.5.1](https://docs.percona.com/proxysql/2.5.1.html)| A high performance, high-availability, protocol-aware proxy for MySQL.          |
+| Percona Toolkit         | [3.5.2](https://www.percona.com/doc/percona-toolkit/LATEST/release_notes.html#v3-5-1-released-2022-11-28)     | The set of scripts to simplify and optimize database operation. |
```

Comment:
Why does the link point to 3.5.1?
https://docs.percona.com/percona-toolkit/release_notes.html#v3-5-2-released-2023-03-28